packages/core/src/login/webview/vue/login.vue

```diff
@@ -566,8 +566,7 @@ export default defineComponent({
 }
 
 .h4 {
```

Comment:
We shouldn't call this `.h4` since this is easily confused with the `h4` element. Can we call it `subHeader` or something.

Reply:
renamed